src/databases/lowdb.js

```diff
@@ -9,7 +9,7 @@ const db = low(adapter);
 
 db.defaults({processedEvents: []}).write();
 
-export function saveParsedEvent(event: Event): Promise<any> {
+export function saveParsedEvent(event: Event): Promise<boolean> {
     logger.debug('Saving %d to lowdb', event.id);
```

Comment:
These lines (the ones calling `logger`) still need to be covered with types, but it's actually not that relevant, so I guess we could focus on that later.